### PR TITLE
A few pretty useful extra commands

### PR DESCRIPTION
**Pull request by d0vgan**
"Edit: Clear Undo Stack" is pretty useful when you don't want to be confused by any Undos from the saved session.
"Show White Spaces: All" and "Show White Spaces: In Selection" are absolute must-have that must have been officially added already. (I aslo considered "Show White Spaces: None" as a third option, but realized that in practice it is much more handy to have just two options to switch between them).

### CODE REVIEW

File: Default.sublime-commands [hunk 1]
```diff
@@ -459,4 +459,9 @@
     { "caption": "Sublime Merge: File History", "command": "sublime_merge_file_history" },
     { "caption": "Sublime Merge: Folder History", "command": "sublime_merge_folder_history" },
     { "caption": "Sublime Merge: Blame File", "command": "sublime_merge_blame_file" },
+
+    // Extra
+    { "caption": "Edit: Clear Undo Stack", "command": "clear_undo_stack" },
+    { "caption": "Show White Spaces: In Selection", "command": "show_white_spaces", "args": {"options": "selection"} },
+    { "caption": "Show White Spaces: All", "command": "show_white_spaces", "args": {"options": "all"} },
```
Comment (deathaxe) [inline Default.sublime-commands]: How about using the builtin `set_setting` command?

```suggestion
    { "caption": "Show White Spaces: In Selection", "command": "set_setting", "args": { "setting": "draw_white_space", "value": "selection" } },
    { "caption": "Show White Spaces: All", "command": "set_setting", "args": { "setting": "draw_white_space", "value": "all" } },
```

**Comment by d0vgan** [comment]
By the way, where did you learn about the `set_setting` command?
I tried to find the complete list of Sublime Text's commands with descriptions - and did not succeed.
The closest I could find was a Command Browser plugin. It allows to look at commands one by one, whereas I'd prefer to see the full list with descriptions as one document.

**Comment by michaelblyons** [comment]
It's not quite what you're looking for, but you may be interested in https://packagecontrol.io/packages/CommandsBrowser

**Comment by deathaxe** [comment]
I took it from PackageDev's autocompletion module.

**Review by deathaxe — APPROVED** [review]

**Pull request merged**